### ISSUE TITLE
GTM integration fails if another script messes with the `form` global variable.

### ISSUE THREAD

**Issue by nitriques** [solspace/craft3-freeform#1549]
### What happened?

We get many many reports in Sentry regarding this error

`null is not an object (evaluating 'form.freeform')`

It is coming from the GTM integration.

![image](https://github.com/user-attachments/assets/53417c84-61d1-4c27-ad3b-57f0f72ef51f)

### Errors and Stack Trace (if available)

```shell
null is not an object (evaluating 'form.freeform') 

When evaluating ar pushEvent = form.freeform._dispatchEvent(
```

### How can we reproduce this?

1. Enable GTM integration for all forms.
2. Load a page with a form with GTM enabled.
3. Before submitting the form, open the console and run `form = null` (which _any_ other script can do).
4. After submit, we get an error.

Adding an iife around the whole block would solve the scoping issue.

```js
(function () {
    // code here
})();
```

### Freeform Edition

Pro

### Freeform Version

5.6.2

### Craft Version

4.12.4.1

### When did this issue start?

After upgrading from older Freeform version, After upgrading from older Craft version

### Previous Freeform Version

from latest 4.x, updated last week

**Comment by gustavs-gutmanis** [comment]
Thank you @nitriques for reporting this, we'll address this issue in our next update.

**Comment by kjmartens** [comment]
This is resolved in Freeform 5.6.6+ now. 🙂

**Comment by nitriques** [comment]
Thanks @kjmartens !
